components/Amendment.js

```diff
@@ -15,18 +14,23 @@ export default function Amendment({
   belongsToCircle = false,
 }) {
   const [, setActiveAmendment] = useGlobal("activeAmendment");
-  const [activeRevision, setActiveRevision] = useGlobal("activeRevision");
+  const [, setActiveRevision] = useGlobal("activeRevision");
+  const [activeCircle] = useGlobal("activeCircle");
 
   const goToRevision = () => {
-    setActiveRevision(amendment.revision.id);
-    RootNavigation.navigate("editRevision");
+    setActiveRevision(amendment.revision.id, () => {
+      RootNavigation.navigate("editRevision", {
+        revision: amendment.revision.id,
+        circle: activeCircle,
+      });
+    });
   };
 
-  useEffect(() => {
-    if (activeRevision) {
-      RootNavigation.navigate("viewRevision");
-    }
-  }, [activeRevision]);
+  // useEffect(() => {
+  //   if (activeRevision) {
+  //     RootNavigation.navigate("viewRevision", { revision: activeRevision });
+  //   }
+  // }, [activeRevision]);
```

Comment:
Pretty sure we don't want a user to just be viewing an amendment on Constitution and then get chucked over to revisions. Ideally we want to use the correct react pattern, and do it this way, but I haven't worked through the process of unsetting activeRevision | activeAmendment |activeChannel on navigation changes